cdc/owner/changefeed.go

```diff
@@ -433,7 +432,7 @@ func (c *changefeed) asyncExecDDL(ctx cdcContext.Context, job *timodel.Job) (don
 	return done, nil
 }
 
-func (c *changefeed) updateStatus(barrierTs model.Ts) {
+func (c *changefeed) updateStatus(ctx cdcContext.Context, barrierTs model.Ts) {
```

Comment:
```suggestion
func (c *changefeed) updateStatus(current model.Ts, barrierTs model.Ts) {
```

Please only pass necessary information.

Reply:
I wil do this lately.

Reply:
done